src/ops.jl

```diff
@@ -109,3 +109,13 @@ function _maxpool_with_inds(t::Tensor{T,M}, pdims::PoolDims{N,K,S,P,D};
 
   Tensor{T,M}(ptr[1], on(t)), Tensor{T,M}(ptr[2], on(t))
 end
+
+function _chunk(t::Tensor{T,N}, chunks=2, dims=1) where {T,N}
+  ts = [t.ptr for _ in 1:chunks]
```

Comment:
Should be initialized as `Vector{Ptr{Cvoid}}` and not use the pointer out of the tensor